concourse/pipelines/deploy.yml

```diff
@@ -420,25 +430,43 @@ jobs:
     plan:
     - in_parallel:
       - get: govuk-infrastructure
-        resource: govuk-infrastructure-frontend
+        resource: govuk-infrastructure-concourse-tasks
         trigger: true
-      - get: govuk-terraform-outputs
+      - get: app-terraform-outputs
+        resource: frontend-terraform-outputs
         passed:
         - run-terraform
         trigger: true
       - get: release
         resource: frontend
         trigger: true
-    - task: update-task-definition
-      file: govuk-infrastructure/concourse/tasks/update-task-definition.yml
-      params:
-        APPLICATION: frontend
-        GOVUK_ENVIRONMENT: test
-    - task: update-ecs-service
-      file: govuk-infrastructure/concourse/tasks/update-ecs-service.yml
+      - task: update-draft-task-definition
+        file: govuk-infrastructure/concourse/tasks/update-task-definition-v2.yml
+        output_mapping:
+         task-definition-arn: draft-task-definition-arn
+        params:
+         APPLICATION: frontend
+         VARIANT: draft
+    - task: update-live-task-definition
+      file: govuk-infrastructure/concourse/tasks/update-task-definition-v2.yml
+      output_mapping:
+       task-definition-arn: live-task-definition-arn
       params:
-        ECS_SERVICE: frontend
-        GOVUK_ENVIRONMENT: test
+       APPLICATION: frontend
+       VARIANT: live
```

Comment:
This looks like it needs to be outdented by two more spaces, so it's within the `in_parralel` block that starts on line 431.